data/devices/data-parse-test.py

```diff
@@ -55,6 +55,10 @@ def check_match_str(string):
         pid = parts[2]
         assert(pid == '{:04x}'.format(int(pid, 16)))
```

Comment:
adding this here because I can't comment on the commit message in the review interface: please use present tense for commit messages. Think of the patch doing something, i.e. "add foo to bar"  (not ~~added foo to bar~~)

Reply:
ok I'll change the name in the rebase